nextflow.config

```diff
@@ -29,29 +54,6 @@ params {
     bracken_length = null
     bracken_level = 'S'
     database_set = "Viral"
-    database_sets = [
```

Comment:
I think it would be good to put this back in but with the climb shared directories provided instead as well as some sort of mapping to the matching kraken2-servers already running? Probably not strictly necessary right now but could be cool

Reply:
Also host our own equivalents on public S3 buckets so that it's easy to run locally since you said that was something you wanted to do

Reply:
added back in main, and there is a climb.config so we can update with local ones when we are ready